graphql-latest.gemfile

```diff
@@ -2,6 +2,6 @@ source "https://rubygems.org"
 
 gem 'coveralls'
 
-gem "graphql", "~> 1.8"
+gem "graphql", ">= 1.8"
```

Comment:
I literally made it as `latest`, allowing it to be `2.x` so that the tests can keep up with the latest graphql-ruby.

Maybe you might prefer it to be `~> 1.8` so that it won't be expanded as `2.x` - let me know what would you think!